dev/package.json

```diff
@@ -37,6 +37,11 @@
         "@khanacademy/wonder-blocks-tooltip": "catalog:",
         "@khanacademy/wonder-stuff-core": "catalog:",
         "@phosphor-icons/core": "catalog:",
-        "aphrodite": "catalog:"
+        "aphrodite": "catalog:",
+        "react": "catalog:",
+        "react-dom": "catalog:"
```

Comment:
These dips are used in this area but weren't explicit. This helps ESLint out when checking extraneous deps.